problem_builder/templates/html/mcqblock.html

```diff
@@ -1,26 +1,29 @@
-<fieldset class="choices questionnaire">
+<fieldset class="choices questionnaire" aria-describedby="description_{{ self.html_id }}">
   <legend class="question">
     {% if not hide_header %}<h3 class="question-title">{{ self.display_name_with_default }}</h3>{% endif %}
-    <p>{{ self.question|safe }}</p>
+    <p id="description_{{ self.html_id }}">{{ self.question|safe }}</p>
   </legend>
   <div class="choices-list">
     {% for choice in custom_choices %}
-    <div class="choice">
-      <div class="choice-result fa icon-2x"></div>
-      <div class="choice-selector">
-        <input id="choice-{{ self.html_id }}-{{ forloop.counter }}" type="radio"
-          name="{{ self.name }}" value="{{ choice.value }}"
-          {% if self.student_choice == choice.value and not hide_prev_answer %} checked{% endif %}
-        />
-      </div>
-      <label class="choice-label" for="choice-{{ self.html_id }}-{{ forloop.counter }}">
-          {{ choice.content|safe }}
+    <div class="choice" aria-live="polite" aria-atomic="true">
+      <label class="choice-label" aria-describedby="description_{{ self.html_id }} feedback_{{ self.html_id }}
+                                                    choice_tips_{{ self.html_id }}-{{ forloop.counter }}">
+        <div class="choice-result fa icon-2x"
+          data-label_correct="{{ label_correct }}" data-label_incorrect="{{ label_incorrect }}" aria-label=""
+        ></div>
+        <div class="choice-selector">
+          <input id="choice-{{ self.html_id }}-{{ forloop.counter }}" type="radio"
```

Comment:
Nit: we can remove the `id` attribute (it doesn't look like we need it anymore now that the label wraps the input element).